.gitignore

```diff
@@ -19,6 +19,8 @@ dependency-reduced-pom.xml
 build/
 .gradle/
 
+!**/dev-bundle/webapp/VAADIN/build
+
```

Comment:
Do we really need the `dev-bundle` to be included? Shouldn't this be something that is generated/built only locally?

Reply:
If we include the dev bundle, running the demo application locally will not require node/npm download and build.
So, if the aim of the application is to quick start an application to see observability up and running, I would preserve the dev-bundle on the repo.
On the other hand, testing observability kit may have more sense on a production build and this will anyway trigger npm install

Reply:
I'd rather remove the bundle, still we have express-build mode by default now and it'll be generated at the first run.

> On the other hand, testing observability kit may have more sense on a production build and this will anyway trigger npm install

True that, but I wouldn't make it default. Let's keep it as a profile for now!

Reply:
Removed dev-bundle files.
Introduced some tweaks to exclude dev-server artifact from production build, as we directly depend on it and not through the platform dependency

Reply:
> Removed dev-bundle files.

Thanks, let's also remove the ignore line then:

```suggestion
```

> Introduced some tweaks to exclude dev-server artifact from production build, as we directly depend on it and not through the platform dependency

What about adding the `vaadin-dev-server` dependencies in a separate profile, complementary to the production one? So that is always activated when the production profile is not, i.e.
```
<profile>
    <id>development</id>
    <activation>
        <activeByDefault>true</activeByDefault>
    </activation>
    <dependencies>
        <dependency>
            <groupId>com.vaadin</groupId>
            <artifactId>vaadin-dev-server</artifactId>
            <version>${flow.version}</version>
        </dependency>
    </dependencies>
</profile>
```

Reply:
Sounds good :+1:

Reply:
done